plugin/src/test/testProject/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+import com.yelp.codegen.plugin.GenerateTaskConfiguration
+
+buildscript {
+  repositories {
+    maven {
```

Comment:
You should be able to replace with `mavenLocal()` and it will work out of the box.